
This patch handles the fact that generated header files shouldn't be
distributed in a better way than before. It is now possible to run "make
dist" directly after configure, so it is not necessary to run "make"
before "dist".

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

Index: examples/echo/Makefile.am
===================================================================
--- examples/echo/Makefile.am.orig
+++ examples/echo/Makefile.am
@@ -4,7 +4,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/include
 
 noinst_PROGRAMS = echo-server
 
-echo_server_SOURCES = echo-glue.h echo-server.h echo-server.cpp
+echo_server_SOURCES = echo-server.h echo-server.cpp
+nodist_echo_server_SOURCES = echo-glue.h
 echo_server_LDADD = $(top_builddir)/src/libdbus-c++-1.la
 
 echo-glue.h: echo-introspect.xml
@@ -13,8 +14,5 @@ echo-glue.h: echo-introspect.xml
 BUILT_SOURCES = echo-glue.h
 CLEANFILES = $(BUILT_SOURCES)
 
-dist-hook:
-	cd $(distdir); rm -f $(BUILT_SOURCES)
-
 MAINTAINERCLEANFILES = \
 	Makefile.in
Index: examples/properties/Makefile.am
===================================================================
--- examples/properties/Makefile.am.orig
+++ examples/properties/Makefile.am
@@ -4,7 +4,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/include
 
 noinst_PROGRAMS = props-server
 
-props_server_SOURCES = props-glue.h props-server.h props-server.cpp
+props_server_SOURCES = props-server.h props-server.cpp
+nodist_props_server_SOURCES = props-glue.h
 props_server_LDADD = $(top_builddir)/src/libdbus-c++-1.la
 
 props-glue.h: props-introspect.xml
@@ -13,8 +14,5 @@ props-glue.h: props-introspect.xml
 BUILT_SOURCES = props-glue.h
 CLEANFILES = $(BUILT_SOURCES)
 
-dist-hook:
-	cd $(distdir); rm -f $(BUILT_SOURCES)
-
 MAINTAINERCLEANFILES = \
 	Makefile.in
